geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/util/GeoMesaBatchWriterConfigTest.scala

```diff
@@ -66,4 +66,60 @@ class GeoMesaBatchWriterConfigTest extends Specification {
       ret should equalTo(None)
     }
   }
+
+  "fetchMemoryProperty" should {
```

Comment:
In that test, I'd also suggest using each of the supported sizes ('k', 'm', 'g').